package.json

```diff
@@ -32,12 +36,15 @@
     "jquery": "^3.3.1",
     "jquery-ui": "^1.12.1",
     "mini-css-extract-plugin": "^0.4.5",
+    "mocha": "^6.0.2",
     "mustache-loader": "^1.4.1",
     "nunjucks": "^3.1.4",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.4.0",
     "terser-webpack-plugin": "^1.1.0",
+    "ts-node": "^8.0.3",
+    "typescript": "^3.4.2",
```

Comment:
As mentioned above, `ts-node` and `typescript` aren’t needed. If we are going to include them, though, we should match the version with `.pre-commit-config.yaml`.

Reply:
hurray for fewer packages